src/library/Text/styled.js

```diff
@@ -121,7 +128,16 @@ export const TextRoot = styled('p', {
       ...getCommonStyles(as, theme, truncate),
       // 1 - Not normalized because we actually want `##em` as applied value
       // 2 - Must come after commonStyles
-      marginBottom: noMargins ? 0 : theme.Text_marginBottom
+
+      /* TargetX Custom Styles */
+
+      /* Disables default bottom margin on text. */
+      // marginBottom: noMargins ? 0 : theme.Text_marginBottom
+
+      ...(bold && { fontWeight: 'bold' }),
+      ...(fontSize && { fontSize }),
+      ...(lineHeight && { lineHeight }),
+      ...(semiBold && { fontFamily: 'Proxima Nova Semi Bold' })
```

Comment:
I'm not a fan of the `semiBold` prop -- it assumes we have a semi-bold compatible font available AND, worse yet, you're currently hard-coding a font family string "Proxima Nova Semi Bold". Instead, we should simply support `fontFamily` passthrough prop.

Alternatively, I'm open to adding the concept of an "alternative font" directly into the theme and then add an `altFont` prop. That would use the alternative font configured in the theme.